lib/src/front_end/ast_node_visitor.dart

```diff
@@ -863,7 +869,20 @@ class AstNodeVisitor extends ThrowingAstVisitor<void>
 
   @override
   void visitRecordLiteral(RecordLiteral node) {
-    throw UnimplementedError();
+    ListStyle style;
+    if (node.fields.length == 1) {
+      // Single-element records always have a trailing comma.
```

Comment:
You'll also need to check that the field is positional. Records with only a single *named* field don't get a trailing comma unless they split:

```dart
var x = (name: value);
```